sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -401,6 +403,27 @@ public partial class AzureMachineLearningSkill : Azure.Search.Documents.Indexes.
         public System.Uri ScoringUri { get { throw null; } }
         public System.TimeSpan? Timeout { get { throw null; } set { } }
     }
+    public partial class AzureOpenAIEmbeddingSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
+    {
+        public AzureOpenAIEmbeddingSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs) { }
+        public string ApiKey { get { throw null; } set { } }
+        public Azure.Search.Documents.Indexes.Models.SearchIndexerDataIdentity AuthIdentity { get { throw null; } set { } }
+        public string DeploymentId { get { throw null; } set { } }
+        public string ResourceUri { get { throw null; } set { } }
+    }
+    public partial class AzureOpenAIParameters
```

Comment:
Is "Parameters" the suffix @KrzysztofCwalina wants currently? He agreed to this with a very early Key Vault, but later seemed to change his mind and opted for "Options" even if required to, say, go from >= 6 parameters to a property bag. /cc @tg-msft

Reply:
In the upcoming preview release, we are using the same names as those in the Swagger spec. However, we are planning to rename these types in the next release.